infra/aws/terraform/prow-build-cluster/kubernetes.tf

```diff
@@ -40,6 +40,39 @@ module "metrics_server" {
   ]
 }
 
+# AWS Load Balancer Controller (ALB/NLB integration).
+resource "helm_release" "aws_lb_controller" {
+  name       = "aws-load-balancer-controller"
+  namespace  = "kube-system"
+  repository = "https://aws.github.io/eks-charts"
+  chart      = "aws-load-balancer-controller"
+  version    = "1.4.8"
+
+  set {
+    name  = "clusterName"
+    value = module.eks.cluster_name
+  }
+
+  set {
+    name  = "serviceAccount.create"
+    value = "true"
+  }
+
+  set {
+    name  = "serviceAccount.name"
+    value = "aws-load-balancer-controller"
+  }
+
+  set {
+    name  = "serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn"
+    value = module.aws_load_balancer_controller_irsa.iam_role_arn
+  }
+
```

Comment:
I'd also set:

- `resources{}`
- `priorityClassName`
- `image.repository`
- `podDisruptionBudget.minAvailable`

and explicitly manage via Terraform:

- `image.tag`

Reply:
Do you have recommended values for:

I'd also set:

- resources{}
- priorityClassName
- podDisruptionBudget.minAvailable (I guess 1 since we have 2 replicas)

> and explicitly manage via Terraform:
>
> `image.tag`

Do we really need to do so? Doesn't we get that implicitly since we bind the chart version? Or they can keep the same chart version but bump the app version?

Reply:
For resources, try:
```
resources:
  limits:
    memory: 256Mi
  requests:
    cpu: 200m
    memory: 128Mi
```

Reply:
What priority classes do we use in the existing cluster (on GCP)? I'd use one of them if we have some, and skip if we don't.

Reply:
The image tag thing isn't essential. Setting it puts drift control even more in Terraform's hands.

Reply:
On most projects like this, I'd pin image tags.